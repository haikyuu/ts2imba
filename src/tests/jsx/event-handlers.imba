import {build} from "../../index"
import * as imbac from 'imba/compiler'

let tsx-code2 = '''
function App(){
	return (
		<button onClick={()=>console.log("2")}>
		</button>
	)
}
'''

test 'multiple levels closed at once 31' do
	const result = await build tsx-code2
	expect(result.code).toMatchSnapshot!
	const out = imbac.compile(result.code, sourceId: 'sth')
	expect(out.js).toBeDefined()
