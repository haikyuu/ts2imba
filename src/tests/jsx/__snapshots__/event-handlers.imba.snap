// Vitest Snapshot v1

exports[`multiple levels closed at once 31 1`] = `
"tag App
	def render
		<self>
			<button onClick=(do console.log(\\"2\\"))>
"
`;
